### PR TITLE
feat(ui): add dedicated view for errors

### PR DESCRIPTION
**Pull request by userquin**
From this comment: https://github.com/antfu/vite-plugin-inspect/pull/87#issuecomment-1671952546

This PR also fixes bad key combination in `filteredTransforms` loop (Vue warning about duplicated keys):
https://github.com/antfu/vite-plugin-inspect/blob/main/src/client/pages/index/module.vue#L138

We need to figure out how to make the wrap, rn we can only show the line before or after the line.

imagen

### CODE REVIEW

**Comment by userquin** [comment]
removing red bg color...

File: src/client/pages/index/module.vue [hunk 1]
```diff
@@ -135,7 +135,7 @@ getHot().then((hot) => {
           </button>
         </div>
         <div border="b main" />
-        <template v-for="tr of filteredTransforms" :key="tr.name">
+        <template v-for="tr of filteredTransforms" :key="`${tr.name}-${tr.index}`">
```
Comment (sxzz) [inline src/client/pages/index/module.vue:138]: Can it be simplified as 
```suggestion
        <template v-for="tr of filteredTransforms" :key="tr.index">
```

**Review by sxzz — APPROVED** [review]

**Comment by userquin** [comment]
should remove cursor hand when excluded

**Comment by userquin** [comment]
fixing some colors in light theme...

**Comment by userquin** [comment]
LGTM

**Pull request merged**